banish/models.py

```diff
@@ -66,14 +69,52 @@ def is_current(self):
 
     class Meta:
         permissions = (("can_ban_user", "Can Ban User"),)
-        verbose_name = "Banishments"
+        verbose_name = "Banishment"
         verbose_name_plural = "Banishments"
         db_table = 'banishments'
 
 
+class Whitelist(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
You should test what happens to an existing install/database when you remove it from `Banishment`. It should Just Work™ but ...